ci-docker-images/Makefile

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-OCIRUSERNAME ?= spinnaker/everest-ocir-push 
+OCIRUSERNAME ?= spinnaker/everest-ocir-push
 OCIREGISTRY ?= iad.ocir.io
 
-IMAGE ?= iad.ocir.io/oracle/oci-cloud-controller-manager-ci-e2e
-VERSION ?= 1.0.1
+IMAGE ?= iad.ocir.io/spinnaker/oci-kube-ci
+VERSION ?= 1.0.2
```

Comment:
We should remove and fail if `VERSION` is not passed in so that we don’t accidentally push and override an existing version.